pkg/runnerutil/runnerutil.go

```diff
@@ -55,17 +52,11 @@ func GetFullPath(workingDir, path string) string {
 	return filepath.Join(workingDir, path)
 }
 
-// ConvertKVToMap converts {"key":"testKey", "value":"testValue"} to {"testKey":"testValue"}
-func ConvertKVToMap(values []v1alpha1.Var) (map[string]string, error) {
-	result := make(map[string]string, len(values))
-	for _, value := range values {
-		data, err := json.Marshal(value)
-		if err != nil {
-			return nil, err
-		}
-		if err := json.Unmarshal(data, &result); err != nil {
-			return nil, err
-		}
+// ConvertMapToSlice converts {"testKey1":"testValue1","testKey2":"testValue2"} to {"testKey1=testValue1","testKey2=testValue2"}
+func ConvertMapToSlice(values map[string]string) []string {
```

Comment:
Just to confirm, will this support nested maps when we represent in the design doc the vars section?

Reply:
mmh good question, not yet but I will refacto this

Reply:
normal vars are handled in a different way https://github.com/multicloudlab/crossplane-provider-ansible/pull/57/files#diff-81ab5282f8bcb328296decc2ee3d42192cd6ce44d6c6199351136b8b72d71691R223